composer.json

```diff
@@ -1,10 +1,75 @@
 {
-  "name": "su-sws/stanford_publication",
-  "description": "A Drupal feature.",
-  "type": "drupal-custom-module",
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "require": {
-      "composer/installers": "^1.2"
-  }
+    "name": "su-sws/stanford_publication",
```

Comment:
Is four spaces our composer.json standard? I've definitely seen it around but we should talk in a dev chat about preference as everything else is 2 space.

Reply:
Drupal core uses 4 spaces for composer.json & lock. https://git.drupalcode.org/project/drupal/-/blob/9.2.x/.editorconfig#L17